index.d.ts

```diff
@@ -198,6 +204,10 @@ export declare class Account {
   loadMerchantRestrictionSettings(
     merchantUuid: string
   ): Promise<AxiosResponse<RestrictionSettingsData>>;
+  syncWithExternalAccount(integration: string, itemId: number): Promise<AxiosResponse<ExternalAccount>>;
+  updateExternalAccount(
```

Comment:
Is this method needed? Because as I can see it is not used in paywall.

Reply:
We have to offer a way for updating an external account, regardless of whether it's currently used in the paywall or not.
The paywall doesn't use it because of some specificities in the flow - turns out that if we do an update from this method, the LiveLike SDK at the paywall side doesn't reflect that change automatically and still uses your old nickname. In order to use the new nickname, you'd have to make an additional call from that SDK. If we have to make that additional call, then it's better not to use this method at all.

Reply:
Ok then, this can be resolved. Thanks for the detailed explanation ;)